_posts/2018-03-23-cache-breaking-for-faster-web-application-updates.md

```diff
@@ -10,26 +10,28 @@ tags: []
 
 ### The Best of Both Worlds
-What if we could cache the majority of our application code to keep it loading fast, but also get updates to our users the first time they load our application after a deploy? The key to making this possible is to disable client-side caching on the main application file, `index.html` and then aggressively cache all other resources. This keeps our application load time down since `index.html` is nothing more than a [50+ line shell around the application](https://github.com/agrc/deq-enviro/blob/c11865a477be1d5970c457636d9c738df58483e0/_src/index.html). Here's an example of new headers for this file:
+What if we could cache the majority of our application code to keep it loading fast, but also get updates to our users the first time they load our application after a deploy? The key to making this possible is to disable client-side caching on the main application file, `index.html`, and then aggressively cache all other resources. This keeps our application load time down since `index.html` is nothing more than a [50+ line shell around the application](https://github.com/agrc/deq-enviro/blob/c11865a477be1d5970c457636d9c738df58483e0/_src/index.html). 
+
+Here's an example of new headers for this file:
 
 ![screenshot of headers]({{ '/images/index_headers.png' | prepend: site.baseurl }}){: .flex .flex--center .outline }
 
-`Cache-Control: no-cache` directs the browser to always check the server for a new version of this file. The bulk of the code for our applications are built into the JS & CSS files. For these files I added a timestamp to the request (e.g. `/dojo/dojo.js?rel=1519311093828`). This timestamp, implemented via [grunt-cache-breaker](https://github.com/shakyShane/grunt-cache-breaker), is changed each time the application is built. Browsers cache resources by unique URL. So, when the URL to the resource is changed, the browser will instantly request a new version from the server. No more wondering if my users are getting updates!
+"`Cache-Control: no-cache`" directs the browser to always check the server for a new version of this file. The bulk of the code for our applications are built into the JS and CSS files. For these files, I added a time stamp to the request (e.g., `/dojo/dojo.js?rel=1519311093828`). This time stamp, implemented via [grunt-cache-breaker](https://github.com/shakyShane/grunt-cache-breaker), is changed each time the application is built. Browsers cache resources by unique URL. So, when the URL to the resource is changed, the browser will instantly request a new version from the server. No more wondering whether my users are getting updates!
```

Comment:
- Regarding _"The bulk of the code for our applications are built..."_: Is "code" typically singular or plural? (Let me know, and I'll add this issue to the AGRC Style Sheet!)
- Regarding _"Browsers cache resources by unique URL."_: Do we mean "Browsers cache each resource by a unique URL." or "Browsers cache resources by unique URLs."? Again, there's a singular/plural issue here, but I'm not positive how to address it.

Reply:
Code is always singular.

Browsers cache each resource by its unique url is what I would say